pipelines/rj_cvl__osinfo_mongo/tasks.py

```
@@ -0,0 +1,1132 @@
+        print("No temp files to process")
+        return processing_results
+
+    # Step 1: Fetch metadata from BigQuery (1 query for all file_ids)
+    file_ids = list(temp_files_map.keys())
+    metadata_query = f"""
+      SELECT
+        files_id,
+        filename
+      FROM `{metadata_source_table}`
+      WHERE files_id IN UNNEST(@file_ids)
+    """
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)